tests/foreman/cli/test_setting.py

```diff
@@ -285,6 +286,50 @@ def test_positive_update_send_welcome_email(self):
               {'search': 'name=send_welcome_email'})[0]['value']
             assert value == host_value
 
+    @tier1
+    def test_positive_enable_disable_rssfeed(self):
+        """Check if the RSS feed can be enabled or disabled
+
+        :id: 021cefab-2629-44e2-a30d-49c944d0a234
+
+        :steps: Set rss_enable true or false
+
+        :expectedresults: rss_enable is updated
+
+        :caseautomation: automated
+        """
+        for value in ['true', 'false']:
+            Settings.set({'name': 'rss_enable', 'value': value})
+            rss_setting = Settings.list({'search': 'name=rss_enable'})[0]
+            self.assertEqual(value, rss_setting['value'])
+
+    @tier1
+    def test_positive_update_rssfeed_url(self):
+        """Check if the RSS feed URL is updated
+
+        :id: 166ff6f2-e36e-4934-951f-b947139d0d73
+
+        :steps:
+            1. Save the original RSS URL
+            2. Update RSS feed with a valid URL
+            3. Assert the RSS feed URL was really updated
+            4. Restore the original feed URL
+
+        :expectedresults: RSS feed URL is updated
+
+        :caseautomation: automated
+        """
+        orig_url = Settings.list({'search': 'name=rss_url'})[0]['value']
+        # The 'valid_url_list()' func  provides list of URLs,
+        # and we need only one.
+        # And there is always at least one URL, so 0 is safe.
```

Comment:
I'd say the comment is not needed, but why not check both http and https alternative when we have chance?

Reply:
This test is more about the ability to update the URL setting than what kind of URL schema is used, but we can check both alternatives, no problem.